.github/workflows/pull-request-conditionals.yaml

```diff
@@ -85,3 +85,14 @@ jobs:
       flavor: ${{ matrix.flavor }}
       test_type: ${{ matrix.test_type }}
     secrets: inherit # Inherits all secrets from the parent workflow.
+
+  evaluate-package-compliance:
+    needs: run-package-test
+    name: Compliance Evaluation
+    strategy:
+      matrix:
+        flavor: [upstream, registry1]
```

Comment:
🫣 this should now also include `unicorn` with the addition of the unicorn flavor